build/xwalk.gni

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains Crosswalk-feature-related build flags. Users should
+# import it into their build folder by run:
+#   echo 'import("//xwalk/build/xwalk.gni")' > out/your_folder/args.gn'
+# Then run:
+#   gn gen out/your_folder
+
+# Set it false when building content_shell.
+declare_args() {
+  is_build_crosswalk = true
+}
+
+if (is_build_crosswalk) {
+  target_os = "linux"
+  target_cpu = "x64"
+  is_debug = false
+
+  # From src/build/config/features.gni
+  # TODO: Enable nacl for Linux.
```

Comment:
What's currently preventing it from being enabled? Just saying it needs to be enabled without saying why it's not working whereas the gyp version does is unhelpful.